.eslintrc.json

```diff
@@ -1,4 +1,8 @@
 {
+  "globals": {
+    "DD_SERVICE": true,
+    "DD_ENV": true
+  },
```

Comment:
This declaration could be misleading (to both engineers, and ESLINT), because these variables only exist in the frontend, and aren't truly global

Relating to a previous comment, if DD_ENV and DD_SERVICE were injected into `process.env` instead of globally, we would not need to introduce global variables (bad coding practice) into our ESLINT config to begin with.

Reply:
will take this discussion offline

Reply:
TODO: figure a way to declare these variables only in the frontend client directory. Strangely, there is already a `helper.d.ts` file that declares these two variables as strings.